src/cache/messages/data/dictionary/dictionary_fetch.rs

```diff
@@ -136,43 +136,52 @@ impl<D: IntoBytes> MomentoRequest for DictionaryFetchRequest<D> {
 /// this is what you're after:
 /// ```
 /// # use std::collections::HashMap;
-/// # use momento::cache::{DictionaryFetchResponse, DictionaryFetchValue};
+/// # use momento::cache::DictionaryFetchResponse;
 /// # use momento::MomentoResult;
-/// # let fetch_response = DictionaryFetchResponse::Hit { value: DictionaryFetchValue::default() };
+/// # let fetch_response = DictionaryFetchResponse::default();
 /// use std::convert::TryInto;
 /// let item: MomentoResult<HashMap<String, String>> = fetch_response.try_into();
 /// ```
 ///
 /// You can also go straight into a `HashMap<Vec<u8>, Vec<u8>>` if you prefer:
 /// ```
 /// # use std::collections::HashMap;
-/// # use momento::cache::{DictionaryFetchResponse, DictionaryFetchValue};
+/// # use momento::cache::DictionaryFetchResponse;
 /// # use momento::MomentoResult;
-/// # let fetch_response = DictionaryFetchResponse::Hit { value: DictionaryFetchValue::default() };
+/// # let fetch_response = DictionaryFetchResponse::default();
 /// use std::convert::TryInto;
 /// let item: MomentoResult<HashMap<Vec<u8>, Vec<u8>>> = fetch_response.try_into();
 /// ```
 #[derive(Debug, PartialEq, Eq)]
 pub enum DictionaryFetchResponse {
-    Hit { value: DictionaryFetchValue },
+    Hit { value: Value },
     Miss,
 }
 
+impl Default for DictionaryFetchResponse {
```

Comment:
does it make sense to have a default for this?

Reply:
I implemented `Default` for various response types to make the doctests easier to write. In hindsight this will be confusing on the public API. I can back out those changes.

Reply:
Fixed in [4941772](https://github.com/momentohq/client-sdk-rust/pull/315/commits/494177262484f075c5454cf6db2d13e6b1f9ec91)